### ISSUE TITLE
Colorbars of Calendars dark

### ISSUE THREAD

**Issue by ykuendig** [andstatus/todoagenda#107]
With my new Phone and Android 12, the color bars seems to be very dark.
Unfortunatly i did not find a setting or solution to lighten things up

Any Help?

**Comment by yvolk** [comment]
If you mean color bars to the left of events^ these are "calendar events colors
" that are set inside a calendar app, not in a widget settings. See e.g. this thread: https://github.com/andstatus/todoagenda/issues/6

**Comment by ykuendig** [comment]
Yes i know. But with the new android 12 device, the colors are somehow "washed out". See screenshots of the old (android 11) and new device.
![IMG_20220820_223525](https://user-images.githubusercontent.com/12490872/185765212-98520d89-e698-4a4c-8908-84097c1226b4.jpg)
![IMG_20220820_212831](https://user-images.githubusercontent.com/12490872/185765216-ca962e28-43ba-4acb-8ca8-f383f3cf52b8.jpg)

**Comment by yvolk** [comment]
@ykuendig Please look at the "Calendar color" app to change calendar colors, see https://github.com/andstatus/todoagenda/issues/6#issuecomment-2109209786
